expertise/create_dataset.py

```diff
@@ -85,17 +87,59 @@ def deduplicate_publications(self, publications):
                 deduplicated.append(pub)
 
         return deduplicated
+    
+    def get_pub_weight(self, pub=None, venueid=None, weight_specification=None):
+        if not venueid or not weight_specification:
```

Comment:
weight_specification can be empty {} as default so you do not this to do this check.

Reply:
Added in recent commit, but empty [] since the schema is a list of JSON